app/scripts/components/modeldetails.js

```diff
@@ -238,8 +240,8 @@ var exports = (function () {
               //   });
             }
           )
-          .catch(e => {
-            console.log("Failed to get model with id", id, "error", e);
+          .catch(() => {
+            //console.log("Failed to get model with id", id, "error", e);
```

Comment:
This is quite an important catch: this is where we catch timeouts and server responds 403, 404, 500, etc. This should not silently fail. Also, console.log is undesired: this procedure should fail gracefully and the user should be notified.